src/spinners/ScaleLoader.jsx

```diff
@@ -15,7 +15,7 @@ class Loader extends React.Component {
         width: ${this.props.width}px;
         height: ${this.props.height}px;
         margin: ${this.props.margin};
```

Comment:
looks like we need to add `px` here as well.

Reply:
No. margin is as a string (see L 41 below). Also the default value is already set to `2px` and this is mentioned in the documentation, so adding `px` here is not necessary.

Reply:
👍 Sorry missed that.